website/static/js/registrationUtils.js

```diff
@@ -726,34 +726,56 @@ Draft.prototype.registerWithoutReview = function() {
 Draft.prototype.register = function(url, data) {
     var self = this;
 
-    $osf.block();
-    var request = $osf.postJSON(url, data);
-    request
-        .done(function(response) {
-            if (response.status === 'initiated') {
-                window.location.assign(response.urls.registrations);
+    var nodeIds = data.nodesToRegister.map(function (node) {
+        return node.id;
+    });
+
+    var payload = {
+        'data': {
+            'type': 'registrations',
+            'attributes': {
+                'draft_registration': window.contextVars.draft.pk,
+                'registration_choice': data.registrationChoice,
+                'children': nodeIds
             }
-        })
-        .fail(function() {
-            bootbox.alert({
-                title: 'Registration failed',
-                message: language.registerFail,
-                callback: function() {
-                    $osf.unblock();
-                    if (self.urls.registrations) {
-                        window.location.assign(self.urls.registrations);
-                    }
-                },
-                buttons: {
-                    ok: {
-                        label: 'Back to project',
-                    }
+        }
+    };
+
+    if(data.embargoEndDate){
+        payload.data.attributes.lift_embargo = data.embargoEndDate.format('YYYY-MM-DDThh:mm:ss');
+    }
+
+    $osf.block();
+    var request = $.ajax({
```

Comment:
Unsure why, but postJSON will 401 if used here.

Reply:
It was failing because credentials weren't being passed, which is necessary for requests to v2. You can (and should) be able to do this with `postJSON`. Looks like it takes an `isCors` option.